intf_.mod

```diff
@@ -541,7 +540,7 @@ PROCEDURE vers () {
 
 :** val(t,tstart) fills global vii[] to pass values back to record() (called from record())
 VERBATIM
-double val (double xx, double ta) { 
+void val(double xx, double ta) {
```

Comment:
prototype changed because function was not returning any value (and we saw undefined behaviour in some cases).